include/pgduckdb/utility/cpp_wrapper.hpp

```diff
@@ -27,6 +28,11 @@ __CPPFunctionGuard__(const char *func_name, FuncArgs &...args) {
 		}
 	}
 
+	if (pg_es_start != PG_exception_stack) {
```

Comment:
This can definitely use a code comment explaining in what cases this can happen.